function/make.go

```diff
@@ -227,5 +229,6 @@ var expressionType = reflect.TypeOf((*Expression)(nil)).Elem()
 var groupsType = reflect.TypeOf(Groups{})
 var contextType = reflect.TypeOf(EvaluationContext{})
 var timerangeType = reflect.TypeOf(api.Timerange{})
+var evaluationNotesType = reflect.TypeOf(EvaluationContext{})
```

Comment:
Whoops- this was used as an untested convenience method. I just removed it because I couldn't see enough use for it, and it poses a little bit of a challenge since you need to request it as a pointer to be useful, which would probably confuse `MakeFunction`.